extension.js

```diff
@@ -464,15 +487,25 @@ export default class SearchLightExt extends Extension {
     this._release_ui();
     this._remove_events();
 
-    this.mainContainer.ease({
-      opacity : 0,
-      duration : 100,
-      mode : Clutter.AnimationMode.EASE_OUT,
-      onComplete : () => {
-        this._visible = false;
-        this.mainContainer.hide();
-      }
-    });
+    if (this._useAnimations) {
+      this.mainContainer.ease({
+        opacity : 0,
+        scale_x : 0.9,
+        scale_y : 0.9,
+        translation_x : this.width * 0.1 / 2,
+        translation_y : this.height * 0.1 / 2,
+        duration : this._animationSpeed,
+        mode : Clutter.AnimationMode.EASE_OUT,
+        onComplete : () => {
+          this._visible = false;
+          this.mainContainer.hide();
+        }
+      });
+    } else {
+      this.mainContainer.opacity = 0;
+      this._visible = false;
+      this.mainContainer.hide();
+    }
     // this._hidePopups();
 
     Meta.enable_unredirect_for_display(global.display);
```

Comment:
Actually, looking at this again, it may be good to move line 511 of extension.js here into the onComplete() event above after line 502 when animations are enabled. I'm worried it could cause issues if we re-enable unredirection too soon. It should also be after line 507 too if that change happens.

I won't be at my computer until the weekend but can do a quick fix and push it then, or else leave it you if you're willing!